Jenkinsfile

```diff
@@ -60,4 +60,10 @@ node("primary") {
       }
     }
   }
+
+  stage("Publish") {
+    if (isReleaseBranch()) {
+      sh 'yarn run publish'
```

Comment:
This should be `yarn publish`

Reply:
Ahh, I can see why you did this actually... it's because of the weird script setup. Maybe add a comment instead to clarify? Alternatively, maybe use a name other than `publish` for the script?